docs/reference/migration/apis/feature-migration.asciidoc

```diff
@@ -56,92 +56,93 @@ Example response:
       "indices" : [ ]
     },
     {
       "feature_name" : "enrich",
-      "minimum_index_version" : "{version}",
+      "minimum_index_version" : "8100099",
       "migration_status" : "NO_MIGRATION_NEEDED",
       "indices" : [ ]
     },
     {
       "feature_name" : "ent_search",
-      "minimum_index_version" : "{version}",
+      "minimum_index_version" : "8100099",
       "migration_status" : "NO_MIGRATION_NEEDED",
       "indices" : [ ]
     },
     {
       "feature_name" : "fleet",
-      "minimum_index_version" : "{version}",
+      "minimum_index_version" : "8100099",
       "migration_status" : "NO_MIGRATION_NEEDED",
       "indices" : [ ]
     },
     {
       "feature_name" : "geoip",
-      "minimum_index_version" : "{version}",
+      "minimum_index_version" : "8100099",
       "migration_status" : "NO_MIGRATION_NEEDED",
       "indices" : [ ]
     },
     {
       "feature_name" : "kibana",
-      "minimum_index_version" : "{version}",
+      "minimum_index_version" : "8100099",
       "migration_status" : "NO_MIGRATION_NEEDED",
       "indices" : [ ]
     },
     {
       "feature_name" : "logstash_management",
-      "minimum_index_version" : "{version}",
+      "minimum_index_version" : "8100099",
       "migration_status" : "NO_MIGRATION_NEEDED",
       "indices" : [ ]
     },
     {
       "feature_name" : "machine_learning",
-      "minimum_index_version" : "{version}",
+      "minimum_index_version" : "8100099",
       "migration_status" : "NO_MIGRATION_NEEDED",
       "indices" : [ ]
     },
     {
       "feature_name" : "searchable_snapshots",
-      "minimum_index_version" : "{version}",
+      "minimum_index_version" : "8100099",
       "migration_status" : "NO_MIGRATION_NEEDED",
       "indices" : [ ]
     },
     {
       "feature_name" : "security",
-      "minimum_index_version" : "{version}",
+      "minimum_index_version" : "8100099",
       "migration_status" : "NO_MIGRATION_NEEDED",
       "indices" : [ ]
     },
     {
       "feature_name" : "synonyms",
-      "minimum_index_version" : "{version}",
+      "minimum_index_version" : "8100099",
       "migration_status" : "NO_MIGRATION_NEEDED",
       "indices" : [ ]
     },
     {
       "feature_name" : "tasks",
-      "minimum_index_version" : "{version}",
+      "minimum_index_version" : "8100099",
       "migration_status" : "NO_MIGRATION_NEEDED",
       "indices" : [ ]
     },
     {
       "feature_name" : "transform",
-      "minimum_index_version" : "{version}",
+      "minimum_index_version" : "8100099",
       "migration_status" : "NO_MIGRATION_NEEDED",
       "indices" : [ ]
     },
     {
       "feature_name" : "watcher",
-      "minimum_index_version" : "{version}",
+      "minimum_index_version" : "8100099",
       "migration_status" : "NO_MIGRATION_NEEDED",
       "indices" : [ ]
     }
   ],
   "migration_status" : "NO_MIGRATION_NEEDED"
 }
 --------------------------------------------------
+// TESTRESPONSE[s/8100099/\\d+/]
```

Comment:
This is currently not working - I'm not sure what the best way of specifying this doc test is

Reply:
Something this should work:

```
// TESTRESPONSE[s/"minimum_index_version" : "8100099"/"minimum_index_version" : $body.$_path/]
```

The `{version}` placeholder gets inserted for tests and for official docs, but not in previews, but it's in the `x.y.z` format.

Reply:
That did it, thanks!